ralph/skills/caretake/modes/reflect.md

```
@@ -251,20 +222,19 @@ AskUserQuestion(
 
 Routing:
 
-- **"Create issue from findings"**: invoke `Skill("ralph:form", args="thoughts/shared/research/YYYY-MM-DD-retro-[description].md")`.
-- **"Deep-dive a specific pain point"**: ask which one, dispatch `Agent(subagent_type="ralph:codebase-analyzer", ...)`, append findings to the same retro doc as `## Follow-up Investigation`.
+- **"Create issue from findings"**: invoke `Skill("ralph:form", args="thoughts/shared/research/YYYY-MM-DD-reflect-[description].md")`.
+- **"Deep-dive a specific pain point"**: ask which one, dispatch `Agent(subagent_type="ralph:codebase-analyzer", ...)`, append findings to the same reflect doc as `## Follow-up Investigation`.
 - **"Save for later — done"**: STOP.
 
 ## §Step 7: Record outcome (optional)
 
 ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)